README.md

```diff
@@ -1,34 +1,18 @@
-# SolidCable
+# Solid Cable
 
-Solid Cable is a DB-based backend for Action Cable.
+Solid Cache is a database-backed Action Cable adapter that keeps messages in a table and continously polls for updates. This makes it possible to drop the common dependency on Redis, if it isn't needed for any other purpose. Despite polling, the performance of Solid Cable is comparable to Redis in most situations. And in all circumstances, it makes it easier to deploy Rails when Redis is no longer a required dependency for Action Cable functionality.
 
 
 ## Installation
-Add this line to your application's Gemfile:
 
-```ruby
-gem "solid_cable"
-```
-
-And then execute:
-```bash
-$ bundle
-```
+Solid Cable is configured by default in new Rails 8 applications. But if you're running an earlier version, you can add it manually following these steps:
```

Comment:
@dhh I think this might be some copy-pasta unless it is going to be the default in 8.

Reply:
I'm very much intending that we aim for Rails 8 by default 😄. Can you hit me up on david@hey.com? Tried emailing pezza@hey.com.

Reply:
Whoops! That's what i get for not checking my email today! Just replied!